modules/ROOT/pages/clauses/listing-procedures.adoc

```diff
@@ -152,6 +152,21 @@ SHOW PROCEDURES
 |===
 | name | description | mode | worksOnSystem
 
+| "cdc.current"
+| "Returns the current change identifier that can be used to stream changes from."
+| "READ"
+| false
+
+| "cdc.earliest"
+| "Returns the earliest change identifier that can be used to stream changes from."
+| "READ"
+| false
+
+| "cdc.query"
+| "Query changes happened from the provided change identifier."
```

Comment:
I don't understand what "happened from the provided change identifier" means (hence my suggestion may be inaccurate).

```suggestion
| "Query changes observed by the provided change identifier."
label:new[Introduced in 5.13]
```